### ISSUE TITLE
Reconnect interop test is broken.

### ISSUE THREAD

**Issue by madongfly** [grpc/grpc-java#848]
```
./interop-testing/build/install/grpc-interop-testing/bin/reconnect-test-client --use_okhttp=true
Starting test:
io.grpc.StatusRuntimeException: UNKNOWN
	at io.grpc.Status.asRuntimeException(Status.java:428)
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:154)
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:104)
	at io.grpc.testing.integration.ReconnectServiceGrpc$ReconnectServiceBlockingStub.start(ReconnectServiceGrpc.java:127)
	at io.grpc.testing.integration.ReconnectTestClient.runTest(ReconnectTestClient.java:96)
	at io.grpc.testing.integration.ReconnectTestClient.main(ReconnectTestClient.java:123)
Caused by: java.lang.IllegalArgumentException: Unable to find decompressor for message encoding identity
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
	at io.grpc.internal.AbstractStream.setDecompressor(AbstractStream.java:306)
	at io.grpc.internal.AbstractClientStream.inboundHeadersReceived(AbstractClientStream.java:122)
	at io.grpc.internal.Http2ClientStream.transportHeadersReceived(Http2ClientStream.java:106)
	at io.grpc.netty.NettyClientStream.transportHeadersReceived(NettyClientStream.java:113)
	at io.grpc.netty.NettyClientHandler.onHeadersRead(NettyClientHandler.java:202)
	at io.grpc.netty.NettyClientHandler.access$900(NettyClientHandler.java:78)
	at io.grpc.netty.NettyClientHandler$LazyFrameListener.onHeadersRead(NettyClientHandler.java:528)
	at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder$FrameReadListener.onHeadersRead(DefaultHttp2ConnectionDecoder.java:316)
	at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder$FrameReadListener.onHeadersRead(DefaultHttp2ConnectionDecoder.java:265)
	at io.netty.handler.codec.http2.Http2InboundFrameLogger$1.onHeadersRead(Http2InboundFrameLogger.java:54)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader$2.processFragment(DefaultHttp2FrameReader.java:450)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader.readHeadersFrame(DefaultHttp2FrameReader.java:459)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader.processPayloadState(DefaultHttp2FrameReader.java:226)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader.readFrame(DefaultHttp2FrameReader.java:130)
	at io.netty.handler.codec.http2.Http2InboundFrameLogger.readFrame(Http2InboundFrameLogger.java:39)
	at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder.decodeFrame(DefaultHttp2ConnectionDecoder.java:100)
	at io.netty.handler.codec.http2.Http2ConnectionHandler$FrameDecoder.decode(Http2ConnectionHandler.java:293)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.decode(Http2ConnectionHandler.java:336)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:327)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:230)
	at io.netty.channel.ChannelHandlerInvokerUtil.invokeChannelReadNow(ChannelHandlerInvokerUtil.java:83)
	at io.netty.channel.DefaultChannelHandlerInvoker.invokeChannelRead(DefaultChannelHandlerInvoker.java:153)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:157)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:946)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:127)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:510)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:467)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:381)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:353)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:703)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Test failed!
```

**Comment by ejona86** [comment]
"Unable to find decompressor for message encoding identity"

@carl-mastrangelo discovered and fixed this problem with #841. Could you test to see if the issue has gone away?

**Comment by madongfly** [comment]
It was fixed by #841
